jsapp/js/components/processing/singleProcessingContent.scss

```diff
@@ -0,0 +1,70 @@
+@use "scss/_colors";
+@use "./singleProcessingVariables";
+
+.single-processing-content {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 100%;
+  height: 100%;
+}
+
+.single-processing-content__tabs {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  height: singleProcessingVariables.$s-single-processing-tabs;
+}
+
+.single-processing-content__tab {
+  cursor: pointer;
+  color: colors.$kobo-gray-55;
+  position: relative;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+  text-transform: uppercase;
+  padding: 16px 0;
+
+  &:not(:first-of-type) {
+    margin-left: 42px;
+  }
+
+  &:hover {
+    color: colors.$kobo-gray-40;
+  }
+
+  &:active:not(.single-processing-content__tab--active) {
+    transform: translateY(1px);
+  }
+
+  &::after {
```

Comment:
This looks like it's bleeding onto the content:
![image](https://user-images.githubusercontent.com/16762675/135645039-4531144d-fe91-4010-8280-727453347e6c.png)